assets/css/style.css

```diff
@@ -1,5918 +0,0 @@
-/**
```

Comment:
Why this file is deleted? Make sure all the styles are available.

Reply:
this file was committed accidentally

Reply:
If so, then restore the previous version. Deleting the file will delete the main file after merging.